.github/dependabot.yml

```diff
@@ -4,7 +4,9 @@ updates:
   - package-ecosystem: "gradle"
     directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
```

Comment:
If we do weekly, should we increase the number then? I seem to recall that we only do something like 5 at a time, so it feels like we will get behind pretty quickly.

Reply:
A fair point, I've bumped the max open PRs to 10 https://github.com/DataBiosphere/terra-landing-zone-service/pull/140/commits/5e19e229d6f3fd6b60aaaeaa02605e3a89532810